src/seedu/addressbook/Main.java

```diff
@@ -30,6 +31,7 @@
     /** The list of person shown to the user most recently.  */
     private List<? extends ReadOnlyPerson> lastShownList = Collections.emptyList();
 
+    private List<Tagging> taggingList = Collections.emptyList();
```

Comment:
this may be too high level to store the tagging list. which class is more appropriate?